test/e2e/metacluster_test.go

```diff
@@ -0,0 +1,61 @@
+package e2e_test
```

Comment:
Missing copyright header, also what do you think about adding `integration` build tag so these tests won't run locally in the dev environment unless we're explicit about it?

Same comment applies to `e2e_suite_test.go`.

Reply:
It looks like rules_go doesn't play great with the integration build tags. I can add `size="large"` to the bazel config though, will that help?

Reply:
I'm not sure, usually dev environments run test using the `go` command. If there is no better way that's fine, maybe I can play around with it later to see how to exclude these locally.

Reply:
If you run `go test` on an individual package, it won't get pulled in. If you're running the whole suite, I recommend using `bazel test` anyway! It's much faster!